general/views.py

```diff
@@ -1,14 +1,9 @@
-from rest_framework.generics import RetrieveAPIView
+from rest_framework.generics import ListAPIView
 from rest_framework.response import Response
 from .serilalizer import *
 from .models import *
 
 # Create your views here.
-class FormFields(RetrieveAPIView):
-    # permission_classes = [IsAuthenticated]
+class FormFields(ListAPIView):
     serializer_class = JobFormFieldSerializer
-
-    def get(self,request,*ags,**kwargs):
-        fields=JobFormField.objects.all()
-        serializer=JobFormFieldSerializer(fields,many=True)
-        return Response(serializer.data)
\ No newline at end of file
+    queryset=JobFormField.objects.all()
```

Comment:
where is permission to check user is authenticated or not?

Reply:
done